framework/include/constraints/AutomaticMortarGeneration.h

```diff
@@ -152,7 +152,64 @@ class AutomaticMortarGeneration : public ConsoleStreamInterface
    */
   bool onDisplaced() const { return _on_displaced; }
 
-public:
+  /**
+   * @return The nodal normals associated with the provided \p secondary_elem
```

Comment:
The comments are pretty clear, do we usually add all arguments with `@param` or is it optional?